distribution/lib/Standard/AWS/0.0.0-dev/src/S3.enso

```diff
@@ -0,0 +1,30 @@
+from Standard.Base import all
+
+import project.AWS_Credential.AWS_Credential
+import project.Errors.S3_Error
+import project.Internal.Auth
+
+polyglot java import software.amazon.awssdk.services.s3.S3Client
+polyglot java import software.amazon.awssdk.services.s3.model.S3Exception
+
+type S3
+    ## Gets the list of the S3 buckets.
+    @credentials AWS_Credential.default_widget
+    list_buckets : AWS_Credential | Nothing -> Vector Text ! S3_Error
+    list_buckets credentials:(AWS_Credential | Nothing)=Nothing = handle_s3_errors <|
+        client = make_client credentials
+        buckets = client.listBuckets.buckets
+        buckets.map b->b.name
```

Comment:
Shall we return just bucket names or do you think it could be worth to have some `Bucket` type that would hold the name + credential info and then allow access into it.

(Ofc not necessary to be done as part of this initial PR - but wondering what do you think in general about the API returning names vs some Enso types 'bucket descriptors' instead)